_usage/locales.md

```diff
@@ -19,109 +19,117 @@ encoding. Babelfish will automatically translate from the server
+
+PostgreSQL doesn't support the `LIKE` clause on non-deterministic
+collations, but Babelfish supports it for `ci_as` collations.
+Babelfish doesn't support `LIKE` on AI collations. Pattern matching
 operations on non-deterministic collations are also not supported.
 
 To establish Babelfish collation behavior, set the following
 parameters :
 
 | Parameter             | Description                                   |
 | --------------------- | --------------------------------------------- |
-| `default_locale`      | The ` default_locale ` parameter is used in combination with the collation attributes in the table above to customize collations for a specific language and region. The default value is `en-US`. The default locale applies to all Babelfish collations and to all SQL Server collations that are mapped to Babelfish collations. This parameter may be changed after initial Babelfish database creation time, but it won\'t affect the locale of existing collations.            |
-| `server_collation_name`  | The collation used as the default collation at both the server level and the database level. The default value is `sql_latin1_general_cp1_ci_as`. You can choose from the collations in the table that follows for the Collation name field when you create your PostgreSQL cluster for use with Babelfish. Don\'t modify the `server_collation_name` after the Babelfish database is created.  |
+| `server_collation_name`  | The collation used as the default collation at both the server level and the database level. The default value is `sql_latin1_general_cp1_ci_as`. When you create your PostgreSQL cluster for use with Babelfish, you can choose the &ldquo;Collation name&rdquo; from the following table. Don't modify `server_collation_name` after the Babelfish database has been created.  |
+| `default_locale`      | The `default_locale` parameter is used to customize the collation chosen with `server_collation_name` for a specific language and region. The default value is `en-US`.  This parameter is ignored if the chosen collation already implies a locale.  This parameter may be changed after initial Babelfish database creation time, but it won't affect the locale of existing collations. |
 
-Use the following collations as a server collation or an object
-collation:
+The following collations (and versions with different modifier suffixes, like `ci_as`) can be used in `server_collation_name` or for column definitions:
 
-| Collation ID          | Notes                                         |
+| Collation Name        | Notes                                         |
 | --------------------- | --------------------------------------------- |
-| `BBF_Unicode_General_CI_AS` | Supports case-insensitive comparison and the `LIKE` operator. |
```

Comment:
these CI_AS collations should not be removed, and are the ONLY collations that may be specified as the server_collation_name.  CS_AS collations may not be specified (in TSQL this would change how identifiers get parsed)

Reply:
I don't understand. The collation is in the list as `bbf_unicode_general_cs_as`, and (cf. the above comment) I said that variants with `ci_as` also work.
My experiments and code scrutiny showed that *all* these collations can be used as `server_collation_name`. Am I wrong there?

Reply:
yes, I think you are wrong there.  In TSQL, unlike in the SQL standard or PostgreSQL, identifiers are parsed using the database collation, and delimited identifiers, as defined by the standard, are not supported.  The net effect of this is that a ci_as collation will treat identifiers abc, aBc, "aBC", and "Abc" as all the same identifier, but for a cs_as database collations, these would each be different identifiers.  Most of the infrastructure is in place to support a cs_as collation, but we haven't tested it yet so we prohibit it.  The code that prohibits it is in the babelfishpg_tsql extension, where the GUC is set.  If there is some codepath that permits the server_collation_name to be set to a cs_as collation, that should be reported as a bug.